src/scss/components/_verification-badge.scss

```diff
@@ -0,0 +1,26 @@
+.lg-verification-badge {
+
+  background-color: map-get($site-palette, 'site-white');
+  border: units(1px) solid map-get($site-palette, 'site-green');
+  border-radius: $theme-border-radius-xxl;
+  box-sizing: border-box;
+  display: inline-block;
+  padding: units(1) units(2);
```

Comment:
USWDS has some globally available spacing units so I swapped the space variables to these. https://v2.designsystem.digital.gov/design-tokens/spacing-units/

Reply:
ah nice!